src/sql/migrations.rs

```diff
@@ -785,6 +785,57 @@ CREATE INDEX msgs_status_updates_index2 ON msgs_status_updates (uid);
         .await?;
     }
 
+    if dbversion < 106 {
+        // Recreate `config` table with UNIQUE constraint on `keyname`.
+        sql.execute_migration(
+            r#"
+CREATE TABLE new_config (
+  id INTEGER PRIMARY KEY,
+  keyname TEXT UNIQUE,
+  value TEXT NOT NULL
+);
+INSERT OR IGNORE INTO new_config SELECT
+  id, keyname, value
+FROM config;
+DROP TABLE config;
+ALTER TABLE new_config RENAME TO config;
+CREATE INDEX config_index1 ON config (keyname);
+"#,
+            106,
+        )
+        .await?;
+    }
+
+    if dbversion < 107 {
+        sql.execute_migration(
+            "CREATE TABLE new_keypairs (
+               id INTEGER PRIMARY KEY AUTOINCREMENT,
```

Comment:
Btw, if the contact is only forward-verified, the green checkmark can already be displayed (we already verified it, so why not display that?) and we even can add the contact to verified chats, but before adding we should retry sending `v*-request-with-auth` because maybe it was lost and that's the reason of the absent backward verification.